integration_tests/test_math.py

```diff
@@ -60,10 +60,9 @@ def test_exp():
 
 
 def test_pow():
-    i: f64
-    i = pow(2.4, 4.3)
-    assert abs(i - 43.14280115650323) < eps
 
+    assert abs(pow(2.4, 4.3) - 43.14280115650323) < eps
+    assert abs(pow(2, 4) - 16) < eps
```

Comment:
What about using variables as input, expressions with multiply, divide, etc, recursive calls? Tests for `fabs` are better in this regard. Please do something similar for `pow` as well.

Reply:
Yes I will. I was unable to narrow down to the error as it kept giving ```Segfault: Signal SIGSEGV (segmentation fault) received``` error. on using the --show-asr command to debug as you had instructed

Reply:
I'll update this with more robust tests and once this pr is merged, I'll make another pr with error files for pow.

Reply:
This is done

Reply:
Well the tests are still not robust enough. But anyways, I have approved it. Waiting for @certik to review.